src/Sulu/Bundle/ContentBundle/Resources/config/forms/PageExcerpt.xml

```diff
@@ -26,6 +26,12 @@
             <title lang="en">Categories</title>
         </meta>
     </property>
+    <property name="tags" type="tag_list">
```

Comment:
`tag_selection`

Reply:
Renaming the content type results in a lot of test fails, so I would prefer to do that in a separate PR. There is already the #3624 tagged on the 2.0 release, so that we don't forget about it.